lib/dockerize_rails/constants.rb

```diff
@@ -31,20 +31,28 @@ module Constants
           :'--test-env' => 'generates configurations to run tests.'.freeze
         }
       },
-      docker_info: {
-        aliases: %I[docker_info di],
-        help: 'Shows Docker information'.freeze
-      },
       undockerize: {
         aliases: %I[undockerize ud du u dd],
         help: 'Removes docker configurations'.freeze,
         params: {
           :'--purge' => "also removes #{Constants::DOCKERIZE_RAILS_CONFIG_FILE_NAME}".freeze
         }
       },
+      docker_info: {
+          aliases: %I[docker_info di],
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

<!-- Comment by [SideCI](https://sideci.com) -->